main.go

```diff
@@ -80,7 +80,7 @@ func releaseToStream(releaseArch *ReleaseArch, release Release) StreamArch {
 
 		if releaseArch.Media.Gcp != nil && releaseArch.Media.Gcp.Image != nil {
 			gcpImage := StreamCloudImage{}
-			gcpImage.Image = fmt.Sprintf("projects/fedora-cloud/global/images/family/fedora-coreos-%s", release.Stream)
+			gcpImage.Image = fmt.Sprintf("projects/fedora-coreos-cloud/global/images/family/fedora-coreos-%s", release.Stream)
```

Comment:
Not familiar with this: I assume this has to correspond to existing GCP projects? Does the `fedora-coreos-cloud` project exist?

Reply:
Correct.  Both `fedora-cloud` and `fedora-coreos-cloud` exist; we've taken the decision to upload FCOS into its own project.